src/webots/gui/WbUpdatedDialog.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright 1996-2021 Cyberbotics Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef WB_UPDATE_DIALOG_HPP
+#define WB_UPDATE_DIALOG_HPP
+
+//
+// Description: Updated dialog for the first launch of Webots 2022a.
+//
+
+#include <QtWidgets/QDialog>
+#include <QtCore/QVariant>
+#include <QtWidgets/QApplication>
+#include <QtWidgets/QPushButton>
+#include <QtWidgets/QGroupBox>
+#include <QtWidgets/QLabel>
+
+class WbUpdatedDialog : public QDialog {
+  Q_OBJECT
+public:
+  explicit WbUpdatedDialog(QWidget *parent = NULL);
+  virtual ~WbUpdatedDialog() {}
+
+private:
+  QLabel *image;
+  QLabel *labelWelcome;
+  QLabel *labelThanks;
+  QLabel *labelNote;
+  QLabel *labelChangelog;
+  QGroupBox *groupBox;
+  QPushButton *closeButton;
```

Comment:
```suggestion
  QLabel *mImage;
  QLabel *mLabelWelcome;
  QLabel *mLabelThanks;
  QLabel *mLabelNote;
  QLabel *mLabelChangelog;
  QGroupBox *mGroupBox;
  QPushButton *mCloseButton;
```
Webots uses the `m` prefix for member variables.